src/index.ts

```diff
@@ -30,14 +37,23 @@ export class WebRTCPlayer extends EventEmitter {
   private adapter: Adapter;
   private statsInterval: any;
   private statsTypeFilter: string;
+  private msStatsInterval: number = 5000;
+  private detectMediaTimeout: boolean = false;
+  private mediaTimeoutThreshold: number = 30000;
+  private timeoutThresholdCounter: number = 0;
+  private bytesReceived: number = 0;
 
   constructor(opts: WebRTCPlayerOptions) {
     super();
     this.videoElement = opts.video;
     this.adapterType = opts.type;
     this.adapterFactory = opts.adapterFactory;
     this.statsTypeFilter = opts.statsTypeFilter;
+    this.detectMediaTimeout = opts.detectTimeout;
 
+    if(opts.timeoutThreshold){
```

Comment:
A nice way to write a one-liner for these sorts of things is to use the [nullish coallescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)
`this.mediaTimeoutThreshold = this.timeoutThreshold ?? this.mediaTimeoutThreshold`

Should be done for `this.detectMediaTimeout` as well since right now it will become `undefined` if the user don't provide that option.